### ISSUE TITLE
Breaking Prisma v2.9.0 Deploy Error: "Query engine binary not found"

### ISSUE THREAD

**Issue by thedavidprice** [redwoodjs/redwood#1345]
@peterp 

**I've upgraded `main` to Prisma v2.9.0.** I was hoping to include it in v0.20.0 as a fix for the build error on Node.js v12.19.0 See https://github.com/redwoodjs/redwood/issues/1313 Gah, catch-22!

I tested this via canary upgrade and deploy on my test project https://github.com/redwoodjs/redwood/pull/1344

Issue confirmed in another deploy test at Redwood v0.19.3 with Prisma packages manually added at v2.9.0.

## Error
Prisma `v2.9.0` is giving me a new deploy error on Netlify regarding "Query engine binary not found":
```
Error: Invalid `prisma.post.findMany()` invocation in /var/task/src/api/dist/services/posts/posts.js:14:22 Query engine binary for current platform "rhel-openssl-1.0.x" could not be found. This probably happens, because you built Prisma Client on a different platform. (Prisma Client looked in "/var/task/src/node_modules/@prisma/client/runtime/query-engine-rhel-openssl-1.0.x") Files in /var/task/src/node_modules/@prisma/client/runtime: index.d.ts index.js index.js.map You already added the platform "native" to the "generator" block in the "schema.prisma" file as described in https://pris.ly/d/client-generator, but something went wrong. That's suboptimal. Please create an issue at https://github.com/prisma/prisma-client-js/issues/new
```

## Logs and Code and Stuff
Here’s my deployed app:
- `https://redwoodjs-tutorial-test.netlify.app/

And the clean build log on Netlify:
- https://app.netlify.com/sites/redwoodjs-tutorial-test/deploys/5f867eb6769c0d00083596d8

And here’s the `schema.prisma` from my test project, which hasn’t changed since I was successfully deploying on 2.8:
- https://github.com/thedavidprice/redwood-tutorial-test/blob/master/api/prisma/schema.prisma

I couldn’t find any related issues on Prisma GitHub.

**Comment by thedavidprice** [comment]
Workaround fix in v0.20.0 release

**Comment by Tobbe** [comment]
Looks like this will be properly fixed in Prisma 2.11, and no workaround should be needed https://github.com/prisma/prisma/issues/4011#issuecomment-719659159
